src/main/java/gomedic/ui/MainWindow.java

```diff
@@ -158,6 +163,7 @@ void fillInnerParts() {
         activityTable = new ActivityTable(logic.getFilteredActivityListById());
         doctorTable = new DoctorTable(logic.getFilteredDoctorList());
         patientTable = new PatientTable(logic.getFilteredPatientList());
+        patientView = new PatientView(logic.getViewPatient(), logic.getFilteredActivityListById());
```

Comment:
Please use the one get filtered by the time, I think it makes more sense for the appointment filtered by time